security/devices.md

```diff
@@ -66,19 +66,19 @@ Some Android devices support Fingerprint security. Once a 6 or greater
 passcode has been set, Fingerprint ID is absolutely acceptable.
 
 ### Setting Passcode Activation Period
-Passcode activation period is the time after which an idle iPhone or iPad will
+Passcode activation period is the time after which an idle android device will
 require a passcode. This can be set by:
 
-1. Settings -> Security & Location
+1. Settings -> Security
 2. Screen Lock gear icon
-3. Choose a time of _5 minutes_ or less.
+3. Lock after screen timeout -> Choose _Immediately_.
```

Comment:
Is this a change in the security policy?

Reply:
hmm good question. maybe I misunderstood the above settings 

@cromwellryan